scenes/enemies/Sheep/Sheep.tscn

```diff
@@ -210,28 +212,36 @@ libraries = {
 offset_top = -15.0
 offset_bottom = 25.0
 
+[node name="AvoidanceRays" parent="." instance=ExtResource("11_nla65")]
+
+[node name="StuckTimer" type="Timer" parent="."]
+wait_time = 1.25
+one_shot = true
+
+[node name="Blackboard" type="Node" parent="."]
+script = ExtResource("12_oa8ou")
+
 [node name="BeehaveTree" type="Node" parent="." node_paths=PackedStringArray("blackboard")]
 script = ExtResource("5_yme22")
 enabled = false
 actor_node_path = NodePath("..")
 blackboard = NodePath("../Blackboard")
 
-[node name="SelectorComposite" type="Node" parent="BeehaveTree"]
-script = ExtResource("6_5ksd8")
+[node name="What am I doing with my life?" type="Node" parent="BeehaveTree"]
```

Comment:
ROFL with this name.

Anyway I think it's better to remove the spaces and I guess the question mark as well.

WhatAmIDoingWithMyLife